themes/scss/map/attributions.scss

```diff
@@ -0,0 +1,84 @@
+/**
+ *  Map attribution
+ */
+
+@import '../variables/sizes';
+@import '../variables/mixins';
+
+$size: 20px;
+
+.CDB-Attribution {
+  position: absolute;
+  right: 16px;
+  bottom: 16px;
+  width: 0;
+  height: 30px;
+  padding: 5px 30px 5px 10px;
+  border-radius: 3px;
```

Comment:
Properties should be ordered position, right, bottom, width, height, padding, transition, border-radius, font-family, line-height, z-index